src/java/main/org/apache/zookeeper/server/ServerStats.java

```diff
@@ -52,12 +52,15 @@
         public long getNumThrottleEvents();
         public long getNumSlowFsyncEvents();
         public long getFsyncTotalNs();
+        public long getPingRequests();
+        public long getReadRequests();
+        public long getWriteRequests();
     }
-    
+
     public ServerStats(Provider provider) {
         this.provider = provider;
     }
-    
+
```

Comment:
Same here, looks like some kind of save hook might have mangled existing formatting.